public/v4/apps/whisper-asr-webservice.yml

```diff
@@ -0,0 +1,36 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        image: onerahmet/openai-whisper-asr-webservice:$$cap_app_version
+        volumes:
+            - $$cap_appname-data:/var/lib/postgresql/data
+        restart: always
+        environment:
+            POSTGRES_USER: $$cap_pg_user
```

Comment:
cap_pg_user?

Reply:
my bad I keep this during my test I'll fix it